homeassistant/components/switch/__init__.py

```diff
@@ -105,6 +109,16 @@ def today_power_mw(self):
         return None
 
     @property
+    def standby_state(self):
+        """ Is the device on - or in standby. """
+        return None
+
+    @property
+    def sensor_state(self):
+        """ Is the sensor on or off. """
```

Comment:
Okay, I get the standby state. Would it make sense to have it be reflected in the state instead of an attribute? It seems that it is the one measurement that would be important.

The switch is either on, off or in standby. We could overwrite the state property for Switch to be able to reflect this:
```python
    @property
    def state(self):
        """ Returns the state. """
        is_on = self.is_on
        if not is_on:
            return STATE_OFF
        elif self.is_standby:
            return STATE_STANDBY
        return STATE_ON
```

If is_standby defaults to `False`, it will not affect the other switches.

So what does the `sensor_state` do exactly? What date does it reflect?